webapp/src/templates/directives/content_row_list_item.html

```diff
@@ -28,9 +28,9 @@ <h4>
           <i class="fa fa-star primary {{primaryContact ? '' : 'ng-hide'}}" translate translate-attr-title="Primary contact"></i>
           <span>{{heading}}</span>
         </h4>
-        <div class="date {{due ? 'due' : ''}}">
```

Comment:
I believe we have some css about this that can be safely removed now that we're not using this class anymore: 
https://github.com/medic/cht-core/blob/master/webapp/src/css/content-list.less#L34

Reply:
Nice catch! Removed.